bentoml/deployment/utils.py

```diff
@@ -103,3 +107,24 @@ def ensure_docker_available_or_raise():
             'Docker is required for this deployment. Please visit '
             'www.docker.come for instructions'
         )
+
+
+def ensure_api_exists_in_bento_archive_api_lists(apis, api_name, bento_name):
```

Comment:
remove this function now?  or change it to verify list of apis like we discussed

Reply:
We can’t remove it for now, since we also using it in sagemaker. Let’s revisit this when we do sagemaker refactor